ogc/src/main/scala/geotrellis/server/ogc/OgcSourceRepository.scala

```diff
@@ -34,15 +34,24 @@ object OgcSourceRepository {
     case WithName(name)      => _.filter(_.name == name)
     case WithNames(names)    => _.filter(rs => names.contains(rs.name))
     case At(t, _)           => _.filter(_.timeInterval match {
-      case Some(OgcTimeInterval(start, None, _)) => start == t
-      case Some(OgcTimeInterval(start, Some(end), _)) => start <= t && t <= end
+      case OgcTimePositions(list) => list.exists(_ == t)
+      case OgcTimeInterval(start, None, _) => start == t
+      case OgcTimeInterval(start, Some(end), _) => start <= t && t <= end
       // Assume valid OgcLayers with no explicit temporal component are valid
       // at all times so they aren't excluded when optional TIME param is null
-      case None => true
+      case _ => true
```

Comment:
Shouldn't we avoid use of `case _` match everywhere we unpack this ADT? If we use `_` match we wont get exhaustive match errors from the compiler.

Reply:
`_` stands for `OgcTimeEmpty` here.